mucommander-commons-file/src/main/java/com/mucommander/commons/file/protocol/local/LocalFile.java

```diff
@@ -380,6 +383,22 @@ private static void addMountEntries(Vector<AbstractFile> v) {
         }
     }
 
+    /**
+     * Adds Desktop to the given volumes entries if home directory is defined and Desktop folder
+     * is present and is writable (~/Desktop)
+     * @param volumesV the <code>Vector</code> to add mount points to
+     * @param homeFolder  a home folder, can be null
+     */
+    private static void addDesktopEntry(Vector<AbstractFile> volumesV, AbstractFile homeFolder) {
+        if (homeFolder == null) {
+            return;
+        }
+        File desktop = Paths.get(homeFolder.getAbsolutePath(), "Desktop").toFile();
+        if (desktop.exists() && desktop.canWrite() && desktop.isDirectory()) {
+            volumesV.add(FileFactory.getFile(desktop.getAbsolutePath()));
+        }
```

Comment:
how about "eating your own dog food" and use AbstractFile right from the beginning (rather than using Java's File at line 396)?

Reply:
PR updated

Reply:
I've made another minor change - using `desktop` that was retrieved from `homeFolder.getDirectChild("Desktop")` instead of retrieving it one again